app/js/demo-controller.js

```diff
@@ -202,18 +240,16 @@ export class DemoPaywallController {
         response.complete().then(() => {
```

Comment:
IMHO `response.complete` doesn't make conceptual sense. Isn't the whole point of a response that it is a resolved result of an asynchronous request object? I.e. something of the form `request.complete().then(response) => {})`.